README.md

```diff
@@ -116,3 +118,69 @@ export const sellTx = async (): Promise<void> => {
   console.log('Transaction hash:', txHash);
 };
 ```
+
+### Mint example
+```typescript
+import {
+  Environment,
+  Moonshot,
+  MigrationDex,
+  MintTokenCurveType,
+} from '@wen-moon-ser/moonshot-sdk-evm';
+import { JsonRpcProvider, Wallet, Transaction } from 'ethers';
+
+const provider = new JsonRpcProvider('RPC_URL' as string);
+const signer = new Wallet('PRIVATE_KEY', provider);
+
+const mockImg = '...' // base64 image
+
+const moonshot = new Moonshot({
+  signer,
+  env: Environment.TESTNET,
+});
+
+const prepMint = await moonshot.prepareMintTx({
+  name: 'TEST_TOKEN',
+  symbol: 'TEST_TOKEN',
+  curveType: MintTokenCurveType.CONSTANT_PRODUCT_V1,
+  migrationDex: MigrationDex.UNISWAP,
+  icon: mockImg,
+  description: 'TEST_TOKEN',
+  links: [{ url: 'https://x.com', label: 'x handle' }],
+  banner: mockImg,
+  creator: await signer.getAddress(),
+  tokenAmount: '10000000000000',
+});
+
+const deserializedTransaction = Transaction.from(
+  prepMint.transaction,
+).toJSON();
+
+const walletAddress = await signer.getAddress();
+
+const feeData = await provider.getFeeData();
+
+const tx = {
+  ...deserializedTransaction,
+  gasPrice: feeData.gasPrice,
+  from: walletAddress,
+  nonce: await provider.getTransactionCount(walletAddress, 'latest'),
+};
+
+const gasLimit = await provider.estimateGas(tx);
+
+const txResponse = await signer.sendTransaction({
+  ...tx,
+  gasLimit,
+});
+
+const receipt = await txResponse.wait();
+
+if (receipt?.status === 1) {
+  await moonshot.submitMintTx({
```

Comment:
in the API side we only confirm the mintTx for EVM chains does it make sense to call it still submit? if the API endpoint is called submit maybe leave it but this might cause some confusion devs will need to submit like you do above